server/controllers/api_controller.go

```diff
@@ -55,6 +56,7 @@ func (a *APIRequest) getCommands(ctx *command.Context, cmdBuilder func(*command.
 	for _, path := range a.Paths {
 		cc = append(cc, &events.CommentCommand{
 			RepoRelDir: strings.TrimRight(path.Directory, "/"),
+			Flags:      path.ExtraArgs,
```

Comment:
can we call this ExtraArgs too? so we follow the same convention

Reply:
sure thing, done.